akka-testkit-typed/src/main/scala/akka/testkit/typed/FishingOutcome.scala

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (C) 2009-2018 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.testkit.typed
+
+import akka.annotation.DoNotInherit
+
+/**
+ * Not for user extension.
+ *
+ * Instances are available from `FishingOutcomes` in the respective dsls: [[akka.testkit.typed.scaladsl.FishingOutcomes]]
+ * and [[akka.testkit.typed.javadsl.FishingOutcomes]]
+ */
+@DoNotInherit sealed trait FishingOutcome
```

Comment:
Do we need `DoNotInherit` if `FishingOutcome` is sealed? For Java users perhaps?

Reply:
Yep, this one is for all the Java people out there!